front/src/applications/operationalStudies/components/ManageTrainSchedule/PowerRestrictionSelector.tsx

```diff
@@ -10,23 +10,41 @@ import {
   getPowerRestriction,
   getPathfindingID,
 } from 'reducers/osrdconf/selectors';
-import { osrdEditoastApi, LightRollingStock } from 'common/api/osrdEditoastApi';
+import { osrdEditoastApi, RollingStock } from 'common/api/osrdEditoastApi';
```

Comment:
Can you remove the line `/* eslint-disable no-restricted-syntax */` ? Thanks !

Reply:
Done